specification/quantum/data-plane/Microsoft.Quantum/preview/2019-11-04-preview/quantum.json

```diff
@@ -0,0 +1,552 @@
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/blobDetails"
+            }
+          }
+        ],
+        "x-ms-examples": {
+          "Get a SAS token to upload data for a Quantum Workspace": {
+            "$ref": "./examples/storageGet.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/sasUriResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/restError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "jobDetailsList": {
+      "description": "List of job details.",
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/jobDetails"
+          }
+        },
+        "count": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Total records count number."
+        },
+        "nextLink": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Link to the next page of results."
+        }
+      }
+    },
+    "jobDetails": {
+      "description": "Job details.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The job id."
+        },
+        "name": {
+          "type": "string",
+          "description": "The job name. Is not required for the name to be unique and it's only used for display purposes."
+        },
+        "containerUri": {
+          "type": "string",
+          "description": "The blob container SAS uri, the container is used to host job data."
+        },
+        "inputDataUri": {
+          "type": "string",
+          "description": "The input blob SAS uri, if specified, it will override the default input blob in the container."
+        },
+        "inputDataFormat": {
+          "type": "string",
+          "description": "The format of the input data."
+        },
+        "inputParams": {
+          "type": "object",
+          "description": "The input parameters for the job. JSON object used by the target solver. It is expected that the size of this object is small and only used to specify parameters for the execution target, not the input data."
+        },
+        "providerId": {
+          "type": "string",
+          "description": "The unique identifier for the provider."
+        },
+        "target": {
+          "type": "string",
+          "description": "The target identifier to run the job."
+        },
+        "metadata": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
```

Comment:
This means that the metadata values can only be of type string (i.e. not numbers etc.). I assume that is intentional.